pkg/operator/sync.go

```diff
@@ -685,15 +726,20 @@ func (optr *Operator) syncMachineConfigDaemon(config *renderConfig) error {
 			mcdServiceAccountManifestPath,
 		},
 		daemonset: mcdDaemonsetManifestPath,
+		configMaps: []string{
+			mcdKubeRbacProxyConfigMapPath,
+		},
 	}
 
 	// Only generate a new proxy cookie secret if the secret does not exist or if it has been deleted.
-	_, err := optr.kubeClient.CoreV1().Secrets(config.TargetNamespace).Get(context.TODO(), "cookie-secret", metav1.GetOptions{})
-	if apierrors.IsNotFound(err) {
-		paths.secrets = []string{"manifests/machineconfigdaemon/cookie-secret.yaml"}
-	} else if err != nil {
-		return err
-	}
+	/*
+		_, err := optr.kubeClient.CoreV1().Secrets(config.TargetNamespace).Get(context.TODO(), "cookie-secret", metav1.GetOptions{})
```

Comment:
Does this still need to be commented out? If so, we should delete it completely to avoid confusion.